src/phoenix/server/app.py

```diff
@@ -114,6 +118,22 @@ def __init__(self, *, app_config: AppConfig, **kwargs: Any):
         self._app_config = app_config
         super().__init__(**kwargs)
 
+    @cached_property
+    def _web_manifest(self) -> Dict[str, Any]:
+        try:
+            with open(self._app_config.web_manifest_path, "r") as f:
+                return cast(Dict[str, Any], json.load(f))
+        except FileNotFoundError as e:
+            if self._app_config.is_development:
+                logger.warning(
+                    f"Web manifest not found at {self._app_config.web_manifest_path}"
+                )
```

Comment:
why do we need a waring? seems fine to skip

Reply:
removed